torch/distributed/distributed_c10d.py

```diff
@@ -124,6 +124,11 @@ def _export_c_types():
 global _c10d_error_logger
 _c10d_error_logger = _get_or_create_logger()
 
+# There is no clear definition of torch.distributed ops. This helper set allows
+# TorchDynamo to selectively disallow all the distributed ops from the Fx
+# graphs.
+distributed_c10d_ops = set()
```

Comment:
@wconstab Is this too hacky? Problem is there is no easy way to tell what's a torch "op" and whats not.

Reply:
Maybe cc @H-Huang @kwen2501 -- do we expect a function to use the `exception_handler` decorator if and only if it is a c10d op?

Reply:
hah yea its kinda hacky.

But otoh i think it's reasonable to have a set of `dynamo_unsupported_distributed_c10d_ops` and explicitly add them all.  We should just get folks like @H-Huang to agree on the set of ops to initially tag.